webui/react/src/pages/ExperimentDetails/ExperimentVisualization.tsx

```diff
@@ -209,71 +209,92 @@ const ExperimentVisualization: React.FC<Props> = ({ basePath, experiment }: Prop
+    // "Learning Curve" aren't helpful or relevant, so we are hiding them
+    if (experiment.config.searcher.name === ExperimentSearcherName.Custom) {
+      return [
+        {
+          children: (
+            <LearningCurve
+              experiment={experiment}
+              filters={visualizationFilters}
+              fullHParams={fullHParams.current}
+              selectedMaxTrial={filters.maxTrial}
+              selectedMetric={filters.metric}
+              selectedScale={filters.scale}
+            />
+          ),
+          key: ExperimentVisualizationType.LearningCurve,
+          label: 'Learning Curve',
+        },
+      ];
+    } else {
+      return [
+        {
+          children: (
+            <LearningCurve
+              experiment={experiment}
+              filters={visualizationFilters}
+              fullHParams={fullHParams.current}
+              selectedMaxTrial={filters.maxTrial}
+              selectedMetric={filters.metric}
+              selectedScale={filters.scale}
+            />
+          ),
+          key: ExperimentVisualizationType.LearningCurve,
+          label: 'Learning Curve',
+        },
+        {
+          children: (
+            <HpParallelCoordinates
+              experiment={experiment}
+              filters={visualizationFilters}
+              fullHParams={fullHParams.current}
+              selectedBatch={filters.batch}
+              selectedBatchMargin={filters.batchMargin}
+              selectedHParams={filters.hParams}
+              selectedMetric={filters.metric}
+              selectedScale={filters.scale}
+            />
+          ),
+          key: ExperimentVisualizationType.HpParallelCoordinates,
+          label: 'HP Parallel Coordinates',
+        },
+        {
+          children: (
+            <HpScatterPlots
+              experiment={experiment}
+              filters={visualizationFilters}
+              fullHParams={fullHParams.current}
+              selectedBatch={filters.batch}
+              selectedBatchMargin={filters.batchMargin}
+              selectedHParams={filters.hParams}
+              selectedMetric={filters.metric}
+              selectedScale={filters.scale}
+            />
+          ),
+          key: ExperimentVisualizationType.HpScatterPlots,
+          label: 'HP Scatter Plots',
+        },
+        {
+          children: (
+            <HpHeatMaps
+              experiment={experiment}
+              filters={visualizationFilters}
+              fullHParams={fullHParams.current}
+              selectedBatch={filters.batch}
+              selectedBatchMargin={filters.batchMargin}
+              selectedHParams={filters.hParams}
+              selectedMetric={filters.metric}
+              selectedScale={filters.scale}
+              selectedView={filters.view}
+            />
+          ),
+          key: ExperimentVisualizationType.HpHeatMap,
+          label: 'HP Heat Map',
+        },
+      ];
+    }
```

Comment:
**suggestion:** it might be a bit less repetitive to contain just the learning curve and if the searcher is NOT custom, then to add the other 3 in. Something like:
```
const tabs = [ <learning curve config> ];
if (experiment.config.searcher.name !== ExperimentSearcherName.Custom) {
  tabs.push(<parallelcoord>, <scatterplot>, <heatmap>);
}
return tabs;
```